Gordon360/Documentation/Gordon360.xml

```diff
@@ -2324,6 +2369,15 @@
             <param name="username"> The username for the user whose phone is to be updated </param>
             <param name="newMobilePhoneNumber">The new number to update the user's phone number to</param>
         </member>
+        <member name="M:Gordon360.Services.ProfileService.UpdateOfficeLocationAsync(System.String,System.String,System.String)">
+            I am adding this
+            <summary>
+            office location setting
+            </summary>
+            <param name="username"> The username for the user whose phone is to be updated </param>

```

Comment:
"phone number" looks like a mistake here.

Reply:
Thank you! Just changed that